static/app/views/starfish/views/spans/types.tsx

```diff
@@ -56,12 +57,20 @@ export const getDurationChartTitle = (spanOp?: string) => {
   return '--';
 };
 
-export const getThroughputChartTitle = (spanOp?: string) => {
+export const getThroughputChartTitle = (spanOp?: string, throughputUnit?: RateUnits) => {
+  let unitLabel = 'Per Minute';
+  if (throughputUnit === RateUnits.PER_SECOND) {
+    unitLabel = 'Per Second';
+  }
+  if (throughputUnit === RateUnits.PER_HOUR) {
+    unitLabel = 'Per Hour';
+  }
```

Comment:
Would be better to define these somewhere next to `RATE_UNIT_LABELS`!

Reply:
Yup, good call!